idl/github.com/uber/cadence/shared.thrift

```diff
@@ -1468,6 +1468,24 @@ struct DescribeTaskListResponse {
   20: optional TaskListStatus taskListStatus
 }
 
+struct GetTaskListPartitionInfoResponse {
+  10: optional PartitionInfo activityTaskPartition
+  20: optional PartitionInfo decisionTaskPartition
+}
+
+struct PartitionInfo {
+  10: optional bool isScalableTaskList
```

Comment:
Do you need this field ? Every task list has atleast one partition i.e. the root partition.

Reply:
I think you are right. Is it true that all the task list are scalable but some can have only one partition? If so, there is no need for this field.

Reply:
Yes, from a mental model perspective, you can think of all task lists as partitioned task lists and by default, there is only one partition